connect/runtime/src/main/java/org/apache/kafka/connect/runtime/errors/RetryWithToleranceOperator.java

```diff
@@ -63,6 +63,8 @@
         TOLERABLE_EXCEPTIONS.put(Stage.HEADER_CONVERTER, Exception.class);
         TOLERABLE_EXCEPTIONS.put(Stage.KEY_CONVERTER, Exception.class);
         TOLERABLE_EXCEPTIONS.put(Stage.VALUE_CONVERTER, Exception.class);
+        TOLERABLE_EXCEPTIONS.put(Stage.TASK_POLL, org.apache.kafka.common.errors.RetriableException.class);
+        TOLERABLE_EXCEPTIONS.put(Stage.TASK_POLL, org.apache.kafka.connect.errors.RetriableException.class);
```

Comment:
there should be only the connect retriable exception here. the other one is typically thrown by producers/consumers.